distribution/test/itests/test-itests-ddf/src/test/java/ddf/test/itests/catalog/TestCatalogValidation.java

```diff
@@ -1065,21 +920,85 @@ private ValidatableResponse executeOpenSearch(String format, String... query) {
                 .then();
     }
 
-    protected String ingestXmlFromResource(String resourceName) throws IOException {
+    protected String ingestXmlFromResource(String resourceName, boolean shouldFail)
+            throws IOException {
         StringWriter writer = new StringWriter();
         IOUtils.copy(IOUtils.toInputStream(getFileContent(resourceName)), writer);
-        return ingest(writer.toString(), "text/xml");
-    }
-
-    protected String ingestXmlFromResource(String resourceName, boolean checkResponse)
-            throws IOException {
-        if (checkResponse) {
-            return ingestXmlFromResource(resourceName);
+        List<String> ids = new ArrayList<>();
```

Comment:
if the ingest is suppose to fail we will keep running this command until it does fail. In that case we will have generated a list of metacard IDs that should be deleted. It might be worth breaking this out into two methods, one that expects failure and one that expects success.